Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool PAUSED = false;
+    public GameObject pauseUI;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape)) {
+            if (PAUSED) Resume();
+            else Pause();
+        }
+    }
+
+    public void Resume() {
+        pauseUI.SetActive(false);
+        Time.timeScale = 1f;
+        PAUSED = false;
+    }
+
+    void Pause() {
```

Comment:
add explicit access modifier (public)

Reply:
pause does not need to be public.

Reply:
If it isn't public, explicitly state it as private